activesupport/lib/active_support/core_ext/string/inflections.rb

```diff
@@ -28,7 +28,7 @@ class String
   #   'apple'.pluralize(2)         # => "apples"
   #   'ley'.pluralize(:es)         # => "leyes"
   #   'ley'.pluralize(1, :es)      # => "ley"
-  def pluralize(count = nil, locale = :en)
+  def pluralize(count = nil, locale = I18n.locale)
```

Comment:
I am not sure on this, since the idea was to the current locale rather than the default one.